### ISSUE TITLE
Unhandled Exception: Invalid argument(s) in CachedVideoPlayerPlusController.initialize

### ISSUE THREAD

**Issue by naipaka** [OutdatedGuy/cached_video_player_plus#26]
Hello,

Thank you for your hard work in developing the `cached_video_player_plus` package. I have been using it in my project and appreciate its features and functionality.

However, I encountered an issue with version 3.0.0 of the `cached_video_player_plus` package. When I try to initialize the `CachedVideoPlayerPlusController`, the following error occurs:

```
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: Invalid argument(s): Illegal argument in isolate message: object is unsendable - Library:'dart:async' Class: _Future@4048458 (see restrictions listed at `SendPort.send()` documentation for more information)
 <- queue in Instance of 'GetStorage' (from package:get_storage/src/storage_impl.dart)
 <- Instance of 'CachedVideoPlayerPlusController' (from package:cached_video_player_plus/cached_video_player_plus.dart)
 <- field this in CachedVideoPlayerPlusController.initialize.<anonymous closure> (from package:cached_video_player_plus/cached_video_player_plus.dart)
 <- resultPort in Instance of '_RemoteRunner<FileInfo?>' (from dart:isolate)

#0      Isolate._spawnFunction (dart:isolate-patch/isolate_patch.dart:398:25)
#1      Isolate.spawn (dart:isolate-patch/isolate_patch.dart:378:7)
#2      Isolate.run (dart:isolate:285:15)
#3      CachedVideoPlayerPlusController.initialize (package:cached_video_player_plus/cached_video_player_plus.da<…>
Lost connection to device.
```

This issue occurs when I run the example package on both iOS and Android devices. I am using Flutter 3.22.0 and Dart 3.4.0.

I would greatly appreciate it if you could look into this matter and provide a solution.

Thank you.

**Comment by OutdatedGuy** [comment]
@naipaka Thanks for brining this issue to my attention. I can reproduce this issue on from the example app. I don't know how this got slipped, but I'll fix this ASAP.

**Comment by OutdatedGuy** [comment]
@naipaka I have retracted `v3.0.0` and published [v3.0.1](https://pub.dev/packages/cached_video_player_plus/versions/3.0.1) which would fix this issue.

**Comment by naipaka** [comment]
@OutdatedGuy 
Thank you for your prompt response! It has been confirmed that both the example package and the application using the package are working properly.
